vcsutils/webhookparser/webhookparser.go

```diff
@@ -46,9 +46,6 @@ type WebhookParser interface {
 // token    - Token to authenticate incoming webhooks. If empty, signature will not be verified.
 // request  - The HTTP request of the incoming webhook
 func ParseIncomingWebhook(provider vcsutils.VcsProvider, token []byte, request *http.Request) (*WebhookInfo, error) {
-	if request.Body != nil {
-		defer request.Body.Close()
-	}
```

Comment:
Isn't the close important?

Reply:
The reason why this change was done can be found in its commit message https://github.com/jfrog/froggit-go/pull/67/commits/9640c8644571e065d7594f5f956afd094927659a